src/Microsoft.Dnx.Tooling/Restore/NuGet/HttpSource.cs

```diff
@@ -70,8 +70,8 @@ internal class HttpSource
             }
 
             var env = RuntimeEnvironmentHelper.RuntimeEnvironment;
-            var userAgentHeader = $"{UserAgentName}/{env.RuntimeVersion} ({env.OperatingSystem} {env.OperatingSystemVersion})";
-            _client.DefaultRequestHeaders.UserAgent.ParseAdd(userAgentHeader);
+            var userAgentHeader = $"{UserAgentName}/{env.RuntimeVersion}({env.OperatingSystem}{env.OperatingSystemVersion})";
+            _client.DefaultRequestHeaders.TryAddWithoutValidation("user-agent", userAgentHeader);
```

Comment:
Actually, it must be case-sensitive:
https://www.google.com/webhp?sourceid=chrome-instant&ion=1&espv=2&ie=UTF-8#q=http%20header%20case%20sensitive